src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -2323,6 +2327,19 @@ export default class GeFormRenderer extends LightningElement{
 
     }
 
+    get elevateTransactionWarning() {
+        return format(this.CUSTOM_LABELS.gePaymentProcessedWarning, [this.CUSTOM_LABELS.commonPaymentServices]);
+    }
+
+    get showElevateTransactionWarning() {
+        // true when data has already been sent to elevate as indicated by payment status
+        const paymentStatus = this.getFieldValueFromFormState(PAYMENT_STATUS);
+        return paymentStatus &&
+            (paymentStatus === this.PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED
+                || paymentStatus === this.PAYMENT_TRANSACTION_STATUS_ENUM.SUBMITTED
+                || paymentStatus === this.PAYMENT_TRANSACTION_STATUS_ENUM.SETTLED);
```

Comment:
I don't feel super strongly about this so feel free to skip.

I think this would look nicer as a named variable. Maybe like `isPaidTransaction`, `isSuccessfulPayment`, or something.